### DISCUSSION TITLE
Hot to register new phone number

### DISCUSSION THREAD

**Discussion by bilo2000** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#113] [General]
Hi,
i have added new number, it is showing "pending" and is tilling to register my number through Post, how can i make it through your call
is there a sample code

**Comment by gabrieldwight** [comment]
Hi, i think you will need to call SetTwostepverification and RegisterWhatsAppPhoneNumber on the same API call. You will have to wait for the results of two step verification then call register phone number.
